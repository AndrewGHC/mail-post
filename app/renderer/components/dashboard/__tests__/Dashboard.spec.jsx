import React from 'react';
import { shallow } from 'enzyme';

import Dashboard from '../Dashboard';

describe('Dashboard', () => {
  it('matches snapshot', async () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});
