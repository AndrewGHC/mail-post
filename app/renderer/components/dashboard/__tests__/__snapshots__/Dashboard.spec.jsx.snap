// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard matches snapshot 1`] = `
<React.Fragment>
  Dashboard
</React.Fragment>
`;
